core/src/stored/backends/gfapi_device.cc

```diff
@@ -338,20 +342,23 @@ int gfapi_device::d_open(const char* pathname, int flags, int mode)
           switch (device_options[i].type) {
             case argument_uri:
               gfapi_uri_ = bp + device_options[i].compare_size;
-              done = true;
+              break;
+            case argument_logfile:
+              gfapi_logfile_ = bp + device_options[i].compare_size;
+              break;
+            case argument_loglevel:
+              gfapi_loglevel_ =
+                  strtol(bp + device_options[i].compare_size, NULL, 10);
               break;
             default:
+              Mmsg1(errmsg, _("Unable to parse device option: %s\n"), bp);
+              Emsg0(M_FATAL, 0, errmsg);
+              goto bail_out;
               break;
           }
+          done = true;
         }
       }
-
```

Comment:
Was this block intentionally removed?

Reply:
If you look closely, you'll find it in the `switch()`'s default-case